ChangeLog.md

```diff
@@ -98,6 +98,8 @@
     * Add-AzureRmAccount
         - Add position for Credential parameter so the following command is allowed: Add-AzureRmAccount (Get-Credential)
         - Updated parameter sets so the SubscriptionId and SubscriptionName are mutually exclusive
+* RecoveryServices.Backup
```

Comment:
You do not change this file. Rather add the changes in your respective directory. you should have a changelog.md in your RP root folder
Follow the guidelines and update changelog.md
[https://github.com/Azure/azure-powershell/blob/dev/CONTRIBUTING.md#updating-the-change-log](https://github.com/Azure/azure-powershell/blob/dev/CONTRIBUTING.md#updating-the-change-log)